src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,7 +43,7 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-
+  //TODO: Did I define all the variables correctly? 
```

Comment:
not a big deal but TODOs like this are not stuff to do, I would either leave a comment on the PR itself, not in the code, or not write anything because we're gonna check that variables are right anyway.